lightning: delete experimental warning in error resolution (#7849)
pingcap/docs: tidb-lightning/tidb-lightning-error-resolution.md

## Error report

> If TiDB Lightning encounters an error during the import, it outputs a report in both the terminal and its log file when it exits.

Reviewer: ```suggestion
If TiDB Lightning encounters an error during the import, it outputs a summary in the terminal and a log at the end of its log file when it exits.
```

> If TiDB Lightning encounters an error during the import, it outputs an error statistics summary in the terminal and a log at the end of its log file when it exits.

Reviewer 2: ```suggestion
If TiDB Lightning encounters errors during the import, it outputs a statistics summary about these errors in both your terminal and the log file when it exits.
```
